ansible/playbook/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title:
+subtitle:
+---
+
+# Hetzner playbooks
+
+## hetzner-init-context
+
+Initializes local hcloud context.
+
+Parameters:
+
+| Variable | Default | Meaning |
+| --- | --- | --- |
+| hetzner_context_name |  | context name |
+| hetzner_token |  | The token to register with Hetzner. |
+
+```bash
+$ ansible-playbook playbook/hetzner-create-hcloud-server.yml
+```
+
+## create hcloud server
+
+Parameters:
+
+| Variable | Default | Meaning |
+| --- | --- | --- |
+| vm_name |  | Name of the VM to be created at hetzner. |
+| server_type | cx41 | Server type. The list can be obtained using `hcloud server-type list` |
```

Comment:
Default should be `cx31`

Reply:
The default is something that the prompt assigns if we press intro without value. What I did was, if the variable is defined from the command line, Ansible doesn't prompt.

Reply:
What I can do is replace prompts with ansible tasks that assign the default value if it is not defined.

Reply:
As by example we will most of the time use as OS - `centos-7` and type - `cx31`, then defaults are needed. 

Is it possible to show a message when we use prompt to by example propose an example for the VM name (e.g ocp-311, k8s-115, ...) ? @jacobdotcosta